pkg/virtualization/network/virtualnetworkadapter/syntheticnetworkadapter.go

```diff
@@ -46,3 +47,11 @@ func (sna *SyntheticNetworkAdapter) SetMACAddress(mac string) (err error) {
 func (sna *SyntheticNetworkAdapter) Rename(newName string) (err error) {
 	return sna.SetPropertyElementName(newName)
 }
+
+func (sna *SyntheticNetworkAdapter) GetGuestNetworkAdapterConfiguration() (guestConfiguration *GuestNetworkAdapterConfiguration, err error) {
+	wmiGuestConfig, err := sna.GetRelated("Msvm_GuestNetworkAdapterConfiguration")
```

Comment:
Doesnt look like it's closed?

Reply:
This instance is closed in `moc-pkg` PR - https://github.com/microsoft/moc-pkg/pull/481/files#r1774119139